### ISSUE TITLE
a litte problem in fpn code

### ISSUE THREAD

**Issue by foocker** [yongtaoge/RetinaFace.PyTorch#4]
in line 45: p3 = F.interpolate(p4, size=[p4.size(2), p4.size(3)]) + p4
should change to: p4 = F.interpolate(p5, size=[p4.size(2), p4.size(3)]) + p4

**Comment by foocker** [comment]
there is no Context Module, so  has no DCN operator in your RetinaFace Module.
i have  a problem: when training, all prediction bbox and landmark was catted  in one dim, how to get the scale level of them for loss function's forward. thx.

**Comment by YongtaoGE** [comment]
1. yes, you are right!
2. I just couldn't understand your second question.

**Comment by foocker** [comment]
maybe something that i am not understand well, i mean like the two-stage,
all predicted anchors should be get its own level from the FPN after
gathered them together, only this we can write the loss function for
backward.
it seems Retina + FPN have no such worry.

yongtaoge <notifications@github.com> 于2019年8月18日周日 下午7:12写道：
